azavea/raster-foundry
# Feature/lol/s3 uploads

Provides settings and UI to allow users to upload files directly to S3.

This is a fairly tightly scoped PR. Only the direct to S3 upload is provided. We don't check if users are yet allowed to upload or provide a good way to avoid name collision in the bucket.

Connects #7
Connects #19 

To test:
 * Get AWS S3 credentials and provision dev machines using them.
 * Visit site and click the upload imagery button on the left side of the layer browser pane.
 * Open the console and networking tabs so you can see things moving for the next step.
 * Click the "Browse" button and add a few files.
 * You should see progress indicators for your files and a done message when the files complete.
 * If you visit the S3 account you should see your new files have arrived.
